UI/menu/expanding.css

```diff
@@ -8,3 +8,42 @@ li.menu_closed > ul {
 li.menu_open > ul {
 	display: block;
 }
+
```

Comment:
No, main.html explicitly requests this css for menu. We may decide to regroup it later.